pkg/plugins/plugins.go

```diff
@@ -148,13 +148,17 @@ func Get(name, imp string) (*Plugin, error) {
 	if err != nil {
 		return nil, err
 	}
-	for _, driver := range pl.Manifest.Implements {
-		logrus.Debugf("%s implements: %s", name, driver)
-		if driver == imp {
-			return pl, nil
+	if imp != "" {
```

Comment:
In the case of the driver abstraction, there would no longer be a an implementation hard coded at the `volume/drivers/drivers.go` layer when calling `plugins.Get()`.  I think making it `Get(name)` instead makes more sense.  The implements comes by asking the plugin endpoint via Activation what it supports.  The implementation then is used in `GetDriver()` to instantiate an adapter.